cg/src/domains/management/subjects/subjects.controller.js

```diff
@@ -1,4 +1,15 @@
 const SubjectsService = require('./subjects.service');
+const _ = require('underscore');
+
+function coverString(string) {
+  const firstCharacter = string[0];
+  const lastCharacter = string[string.length - 1];
+  let coveredString = new Array(string.length).fill('*');
+  coveredString[0] = firstCharacter;
+  coveredString[string.length - 1] = lastCharacter;
+  coveredString = coveredString.join('');
```

Comment:
coveredString actually becomes a completed different object here (it goes from an array to a string), so we should not re-use the variable 

In the interest of code clarity and readability I would suggest you create a new variable at the stage you are getting the final string, and return that.

Reply:
Using a new variable here also allows the variable above you declared with `let` to be a `const`

Reply:
Hmm I see, it's interesting that I can change a const array after the first assignment, didn't knew that. I really want to comprehend more how Javascript works